client/my-sites/email/email-forwarding/style.scss

```diff
@@ -89,3 +89,9 @@ ul.email-forwarding__list {
 	line-height: 3em;
 	margin-bottom: 15px;
 }
+
+.email-forwarding-add-new-compact-list__actions {
+	flex-direction: row;
+	display: flex;
+	justify-content: space-between;
+}
```

Comment:
I think this is the wrong place for this style block -- this `style.scss` file is only imported by `EmailForwarding`, and is not imported by `EmailForwardingAddNewCompactList.jsx`, so the styles are not going to be applied when the latter component is imported from the email comparison page.